renovate.json

```
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:best-practices", ":disableDependencyDashboard", ":rebaseStalePrs"],
+  "timezone": "Asia/Tokyo",
+  "schedule": ["every weekend"],
+
+  "packageRules": [
+    {
+      "matchManagers": ["github-actions"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "groupName": "github actions deps"
+    },
+    {
+      "matchManagers": ["npm"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "groupName": "minor and patch"
+    },
+    {
+      "matchPackageNames": ["react", "react-dom", "@types/react", "@types/react-dom"],
+      "matchUpdateTypes": ["minor"],
+      "groupName": "react monorepo"
+    },
+    {
+      "matchPackageNames": ["/^eslint/"],
+      "matchUpdateTypes": ["minor"],
+      "groupName": "eslint and config"
+    },
```

わざわざeslint単体で切り出さずに `group: linters` 入れておけば良いかと

https://docs.renovatebot.com/presets-group/#grouplinters

自分だったらここにbiomeも一緒に乗っけると思う

```suggestion
    {
      "extends": ["group: linters"],
      "matchPackageNames": ["@biomejs/biome"]
    },
```

---

`customeManagers:biomeVersions`と`@biomejs/biome`は別々なんですね。
"customManagers:biomeVersions"は設定ファイルのスキーマバージョン管理で"@biomejs/biome"はBiomeのversion管理ですね。